extensions/nns/Cargo.toml

```diff
@@ -46,5 +50,4 @@ pre-release-replacements = [
 ]
 
 [package.metadata.dist]
-include = ["extension.json"]
-
+include = ["extension.json", "sns-cli", "ic-admin", "ic-nns-init"]
```

Comment:
I just played with the idea, but it would be a lot of work (we would need to repackage everything during e2e tests and release CI manually). I've been closely monitoring the cargo-dist project for a while now; as soon as they'll make a release supporting this functionality, I'll bring it here.

Reply:
Sounds good!